packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js

```diff
@@ -150,97 +150,102 @@ export default (async function PgIntrospectionPlugin(
     pgIncludeExtensionResources = false,
   }
 ) {
+
+  //TODO why have all these nested functions
   async function introspect() {
-    // Perform introspection
+
     if (!Array.isArray(schemas)) {
       throw new Error("Argument 'schemas' (array) is required");
     }
+
+    //TODO find out what the cacheing mechanism is doing
```

Comment:
Plugins can cache to speed up startup (e.g. no need to re-introspect the DB); useful for serverless. See https://www.graphile.org/postgraphile/v4-new-features/#cache-invalidation-serverless